client/analytics/report/categories/index.js

```diff
@@ -19,33 +19,37 @@ import CategoriesReportTable from './table';
 import getSelectedChart from 'lib/get-selected-chart';
 import ReportChart from 'analytics/components/report-chart';
 import ReportSummary from 'analytics/components/report-summary';
+import ProductsReportTable from '../products/table';
 
 export default class CategoriesReport extends Component {
 	getChartMeta() {
 		const { query } = this.props;
+		const isCompareView =
+			'compare-categories' === query.filter &&
+			query.categories &&
+			query.categories.split( ',' ).length > 1;
+		const singleCategoryView = 'single_category' === query.filter && !! query.categories;
```

Comment:
What do you think about renaming `singleCategoryView` to `isSingleCategoryView`? That way it would be clearer it's a boolean.